lib/streamlit/elements/lib/policies.py

```diff
@@ -30,7 +30,8 @@
 
 
 def check_callback_rules(dg: DeltaGenerator, on_change: WidgetCallback | None) -> None:
-    """Ensures that widgets other than `st.form_submit` within a form don't have an on_change callback set.
+    """Ensures that widgets other than `st.form_submit` within a form don't have an \
```

Comment:
nit: `st.form_submit` doesn't exist, probably referring to `st.form_submit_button`?